server/api/api.go

```diff
@@ -115,6 +116,17 @@ func (a *API) RegisterAdminRoutes(r *mux.Router) {
 
 func (a *API) requireCSRFToken(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		defer func() {
```

Comment:
The code looks good, but I think this should be an independent middleware. Makes no sense to include the recovery mechanism as part of the CSRF token validation.

Reply:
Ok, I'll create a separate middleeare method.

Reply:
Fixed via https://github.com/mattermost/focalboard/pull/2072/commits/f14ef7f34e15ddd01230adfe3e449d83845093fb